gr-blocks/include/gnuradio/blocks/complex_to_interleaved_char.h

```diff
@@ -39,7 +39,7 @@ class BLOCKS_API complex_to_interleaved_char : virtual public sync_interpolator
     /*!
      * Build a complex to interleaved chars block.
      */
-    static sptr make(bool vector = false);
+    static sptr make(float scale_factor = 1.0, bool vector = false);
```

Comment:
I'm with @mormj here: let's put the new parameter -after- the current one, so that the prior API will still work; just augmenting it.